lms/djangoapps/ccx/utils.py

```diff
@@ -266,12 +268,31 @@ def ccx_students_enrolling_center(action, identifiers, email_students, course_ke
                 errors.append(u"{0}".format(exp))
                 continue
 
-            if CourseEnrollment.objects.is_course_full(ccx_course_overview) and not must_enroll:
-                error = _(u'The course is full: the limit is {max_student_enrollments_allowed}').format(
-                    max_student_enrollments_allowed=ccx_course_overview.max_student_enrollments_allowed)
-                log.info(u"%s", error)
-                errors.append(error)
-                break
+            # Check the License limit in the pearson course operation plugin.
+            if (
+                    is_ccx_course(course_key)
+                    and configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False)
```

Comment:
@elvinquero, let's put the conditional flag first to optimize the execution.
```suggestion
                    configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False)
                    and is_ccx_course(course_key)
```